ml-proto/test/stdio_write.wast

```diff
@@ -0,0 +1,14 @@
+(module 
+  (import $write "stdio" "write" (param i32 i32))
+
+  (memory 4096 4096 (segment 0 "hello, world!\0D\0A\00"))
```

Comment:
The alternative is to be unusable for text strings in C/C++. How about a separate function for binary data?